privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardHybridViewModel.kt

```diff
@@ -221,24 +221,14 @@ class PrivacyDashboardHybridViewModel @Inject constructor(
         this.site.value = site
     }
 
-    private suspend fun createRemoteFeatureSettingsViewState() = withContext(dispatcher.io()) {
-        return@withContext RemoteFeatureSettingsViewState(
```

Comment:
Can we remove `RemoteFeatureSettingsViewState`, `PrimaryScreenSettings`, `LayoutType` classes entirely?

Reply:
Not without changes in the dashboard code, so I'd rather leave this part in for now, we can certainly look to remove if we end up never using any layout params again :)